WordPress/twentytwentyfive
# WIP: Add translation functions to text strings

**Description**
This PR adds translation functions around text strings in patterns.

I have;
- Ran the npm script to add the translation functions.
- Adjusted some strings that the script broke.
- Removed some trailing, duplicate periods in the "no results" texts.
- Updated some strings to use printf.
- Added context to some strings.
- Ran the PHP lint script

To do: 
- [x] Replace the curly single quotes etc with their HTML entities.
- [x] Adjust the split text "Starting at $30 /month" in the "Call to action with grid layout with products and link" pattern
- [ ]  Make the post term separators translatable ``
- One of these blocks are in an HTML file and needs to either be reset to the default value, or be moved to a pattern.

**Testing Instructions**
Review the code changes to find mistakes and things that can be improved further. ❤️ 
Preview and place templates and patterns to make sure there are no block validation errors.
